alexandria/core/models.py

```diff
@@ -239,8 +241,24 @@ def set_checksum(self):
     def save(
         self, force_insert=False, force_update=False, using=None, update_fields=None
     ):
+        validate_file_infection(self.content)
```

Comment:
I'm not too sure about having this level of validation it in the model. Could we move it to a DGAP validator and trigger that from the Python API instead? Of course that would imply applications need to be aware of the DGAP class doing this, but still, I think it'd be worth it for a cleaner codebase

Reply:
Created a base validator class which is passed to dgap per default.